src/mame/drivers/cischeat.cpp

```diff
@@ -1184,13 +1185,15 @@ static INPUT_PORTS_START( bigrun )
 	PORT_DIPSETTING(    0x02, "2 (Green-White Car)" )
 	PORT_DIPSETTING(    0x04, "3 (Red-White Car)" )
 	PORT_DIPSETTING(    0x06, "4 (Yellow Car)" )
-	PORT_DIPUNKNOWN_DIPLOC(  0x08, 0x00, "SW1:4")       /* Always ON according to manual */
+	PORT_DIPNAME( 0x08, 0x00, "Linking" ) PORT_DIPLOCATION("SW1:4")
```

Comment:
"Linking" is ambiguous.
And old note contradicts with your assumption, is it undocumented behaviour or something that the old author missed?

Reply:
Did further testing on DSW1:4 (for bigrun) and still would say "equipped with com-link" if you go into comm test and change the link id dip (maybe due to unemulated linking regs?). The manual itself stages DSW1:4 has to be enabled for linking to work so that's made me conclude this switch would enable linking.

Reply:
So it's "Enable Linking" and it's undocumented behaviour and needs to be addressed.

Reply:
It is undocumented. I'm tempted to revert DSW1:4 back to unknown with the previous note (at least until the link function for these games work).